modules/tinymce/src/core/test/ts/browser/init/ShadowDomEditorTest.ts

```diff
@@ -0,0 +1,93 @@
+import { Pipeline, Step } from '@ephox/agar';
+import { document, setTimeout, StyleSheet } from '@ephox/dom-globals';
+import { TinyLoader } from '@ephox/mcagar';
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+import { ShadowDom, Element, Insert, Body, Remove } from '@ephox/sugar';
+import { UnitTest, Assert } from '@ephox/bedrock-client';
+import { Arr, Futures, Global, Strings } from '@ephox/katamari';
+import { Future } from '@ephox/katamari/lib/main/ts/ephox/katamari/api/Future';
```

Comment:
This shouldn't be imported directly (looks like eslint is failing because of this and other issues).

Reply:
Fixed.